recipe/meta.yaml

```diff
@@ -31,12 +31,12 @@ requirements:
     - impyla >=0.15.0
     - lz4
     - multipledispatch >=0.6
-    - numpy >=1.11
-    - pandas >=0.25
+    - numpy >=1.15
+    - pandas >=0.25.3
     - psycopg2
-    - pyarrow >=0.13
+    - pyarrow >=0.15
```

Comment:
We've got pyarrow 0.13 in the CI builds. Should we update the CI builds, or allow lower versions here?